.github/workflows/apply-version-label-issue.yml

```diff
@@ -0,0 +1,15 @@
+name: Apply version label to issue
+
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  add-version-label-issue:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+
+    steps:
+      - uses: gabrieldonadel/core-workflow-apply-version-label@v0.0.1
```

Comment:
Left a comment here: https://github.com/gabrieldonadel/core-workflow-apply-version-label/commit/82afa877f408c01423b1eeb01781bdb461e9f9e6#r64521057 but can we change this action to be used with a custom issue filter?

Reply:
Sure, I've just released a new version including a `required-label` input to the action and also updated this PR to use the new parameter

Reply:
Here are the tests using `actions-apply-version-label@0.02`
https://github.com/gabrieldonadel/react-native/issues/11
https://github.com/gabrieldonadel/react-native/issues/10